<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveRmaServerOptionsWidget</class>
 <widget class="QWidget" name="ActiveRmaServerOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Active RMA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chkActiveRmaConnection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Accept connections from &quot;active&quot; RMA on TCP port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinActiveRmaPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>5056</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="rdActiveRmaAnyIp">
     <property name="text">
      <string>Accept connections from any IP address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rdActiveRmaFollowingIp">
     <property name="text">
      <string>Accept connections from the following IP addresses</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chkActiveRmaAcceptedConnections">
     <property name="text">
      <string>Record info about accepted connections</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkActiveRmaRejectedConnections">
     <property name="text">
      <string>Record info about rejected connections</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
